Google.Api.CommonProtos/Google.Api.CommonProtos.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Protobuf" Version="[3.18.0, 4.0.0)" />
+    <PackageReference Include="Google.Protobuf" Version="[3.20.0, 4.0.0)" />
```

Comment:
Should this one be 3.21.0 ?? On https://github.com/googleapis/gapic-generator-csharp/pull/487 you said we should move to 3.21.0 before GaxV4

Reply:
Hmm... had thought I'd done that. (It all got a bit confusing due to timing.) Will have a look.